pkg/kapp/config/default_test.go

```diff
@@ -105,14 +106,14 @@ spec:
   0,  3 +   annotations:
   0,  4 +     kapp.k14s.io/versioned: ""
   0,  5 +   name: test-configmap-ver-1
-  0,  6 + 
+  0,  6 +
```

Comment:
I think this is causing the tests to fail. Was this change intended?

Reply:
Ah, not sure why that changed - probably an autoformatting thing with the editor I was using at the time :( - Will try to get that fixed.

Reply:
This should be fixed with the latest push - thanks for catching! 

Lesson learned to not use my editor to do a find + replace and instead use `sed` or equivalent 🤦‍♂️